launcher/screens/Guide/shared.ts

```diff
@@ -0,0 +1,46 @@
+import { StyleSheet } from 'react-native';
+import { useAnimatedStyle, withTiming } from 'react-native-reanimated';
+import { ScaledSizes } from '@metacraft/ui';
+import { HoveredStyleFunc } from 'components/Marketplace/Button/shared';
+import { iStyles } from 'utils/styles';
+
+export const headingSize = [35] as ScaledSizes;
+
+export const useHoveredStyle: HoveredStyleFunc = (isHovered) =>
+	useAnimatedStyle(() => ({
+		opacity: withTiming(isHovered.value ? 0.5 : 1, { duration: 250 }),
+	}));
+
+export const sharedStyle = StyleSheet.create({
+	sectionContainer: {
+		alignItems: 'center',
+		...iStyles.wideContainer,
+	},
+	heading: {
+		fontFamily: 'Volkhov',
+		fontWeight: '600',
+		color: '#fff',
+		textAlign: 'center',
+	},
+	subHeading: {
+		fontFamily: 'Poppins',
```

Comment:
The default font family of `<Text>` component is already `Poppins`, and default color in theme has been changed to `#fff`